easybuild/easyconfigs/p/picard/picard-2.9.2-Java-1.8.0_74.eb

```diff
@@ -0,0 +1,27 @@
+easyblock = 'JAR'
+
+name = 'picard'
+version = '2.9.2'
+
+homepage = 'http://broadinstitute.github.io/picard/'
+description = """A set of tools (in Java) for working with next generation sequencing data in the BAM
+  (http://samtools.github.io/hts-specs) format."""
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+source_urls = ['https://github.com/broadinstitute/%(name)s/releases/download/%(version)s/']
+sources = ['%(name)s.jar']
+
+java = 'Java'
+javaver = '1.8.0_74'
+versionsuffix = '-%s-%s' % (java, javaver)
```

Comment:
use the `%(javaver)s` template, move `versionsuffix` up right below `version`, cfr. https://github.com/hpcugent/easybuild-easyconfigs/blob/master/easybuild/easyconfigs/p/picard/picard-2.1.1-Java-1.8.0_74.eb